server/pulp/plugins/rsync/configuration.py

```diff
@@ -0,0 +1,139 @@
+import logging
+
+_LOG = logging.getLogger(__name__)
+
+
+class OneOfValidation(object):
```

Comment:
All of these objects need docstrings. Maybe not all of the methods, but at least the objects themselves.